modules/vaos/app/services/eps/configuration.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Eps
+  class Configuration < Common::Client::Configuration::REST
+    def self.login_url
```

Comment:
we don't need to define class methods like this, and certainly don't need to define methods for every config setting. we can always do:
`delegate :base_path, :another_config, :yet_another, to: :settings`